example/c_json/pubspec.yaml

```diff
@@ -5,7 +5,7 @@
 name: c_json_example
 
 environment:
-  sdk: '>=2.12.0-198.0.dev <3.0.0'
+  sdk: '>=2.12.0-230.0.dev <3.0.0'
```

Comment:
Should this also be 237? (Also in other places.)

Reply:
Done.

Reply:
This hint for empty structs was added in 230 so I don't think this should of any issue.